src/components/common/DeleteModal.tsx

```diff
@@ -88,3 +107,6 @@ export default function DeleteModal({ open, setOpen, title, description }: Modal
     </>
   );
 }
+function sendRequest() {
```

Comment:
Si no lo uses, no lo pongas

Reply:
ys estaba ahi, aparecio en un merge

Reply:
lo quité, me dio este error:
/home/runner/work/Zeitgeist-FrontEnd/Zeitgeist-FrontEnd/src/pages/Tasks/task-detail.tsx
Warning:   40:6  warning  React Hook useEffect has a missing dependency: 'sendRequest'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

ya lo volvi a poner